### PR TITLE
Support AI.SCRIPTEXECUTE and AI.MODELEXECUTE

### PR DESCRIPTION
**Pull request by AvitalFineRedis**
Add support for  AI.SCRIPTEXECUTE and AI.MODELEXECUTE commands.
Also added support to  `minbatchtimeout` to the model interface (included in ModelGet command).

### CODE REVIEW

File: redisai/commands_test.go [hunk 1]
```diff
@@ -625,29 +643,29 @@ func TestCommand_ModelDel(t *testing.T) {
 	}
 }
 
-func TestCommand_ModelRun(t *testing.T) {
-	keyModel1 := "test:ModelRun:1"
-	keyModel2 := "test:ModelRun:2:Pipelined"
+func TestCommand_ModelExecute(t *testing.T) {
+	keyModel1 := "test:ModelExecute:1"
```
Comment (alonre24) [inline redisai/commands_test.go:647]: Please add documentation to the test

**Review by alonre24 — COMMENTED** [review]
Looks good, what about DAG execute? Are you planning to do it in a separate PR?

**Comment by AvitalFineRedis** [comment]
> Looks good, what about DAG execute? Are you planning to do it in a separate PR?

yes :)

**Review by alonre24 — APPROVED** [review]
Good job :)

**Pull request merged**